src/main/assembly/dist/bin/easy-pid-generator

```diff
@@ -34,27 +35,21 @@ start_jsvc_exec()
 {
     echo "Starting $NAME ..."
     jsvc_exec
-
-    local exitcode=$?
-
-    if [[ $exitcode == 0 ]]; then
+    if [[ $? == 0 ]]; then
```

Comment:
I had considered this version, but I was not sure whether `$?` would change between this line and line 41. Can you guarantee that this does **not** happen?